kie-server-parent/kie-server-controller/kie-server-controller-standalone/src/main/java/org/kie/server/controller/service/StandaloneController.java

```diff
@@ -0,0 +1,382 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+package org.kie.server.controller.service;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLDecoder;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import static org.kie.server.controller.service.ControllerUtils.*;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.kie.server.api.model.KieServerInfo;
+import org.kie.server.controller.api.KieServerControllerException;
+import org.kie.server.controller.api.KieServerControllerNotFoundException;
+import org.kie.server.controller.api.model.KieServerInstance;
+import org.kie.server.controller.api.model.KieServerInstanceInfo;
+import org.kie.server.controller.api.model.KieServerInstanceList;
+import org.kie.server.controller.api.model.KieServerSetup;
+import org.kie.server.controller.api.model.KieServerStatus;
+import org.kie.server.controller.api.model.runtime.Container;
+import org.kie.server.controller.api.model.runtime.ServerInstanceKey;
+import org.kie.server.controller.api.model.spec.Capability;
+import org.kie.server.controller.api.model.spec.ContainerConfig;
+import org.kie.server.controller.api.model.spec.ContainerSpec;
+import org.kie.server.controller.api.model.spec.ContainerSpecKey;
+import org.kie.server.controller.api.model.spec.ContainerSpecList;
+import org.kie.server.controller.api.model.spec.ProcessConfig;
+import org.kie.server.controller.api.model.spec.RuleConfig;
+import org.kie.server.controller.api.model.spec.ServerTemplate;
+import org.kie.server.controller.api.model.spec.ServerTemplateKey;
+import org.kie.server.controller.api.model.spec.ServerTemplateList;
+import org.kie.server.controller.api.service.RuntimeManagementService;
+import org.kie.server.controller.impl.KieServerControllerImpl;
+import org.kie.server.controller.impl.KieServerInstanceManager;
+import org.kie.server.controller.impl.service.RuntimeManagementServiceImpl;
+import org.kie.server.controller.impl.service.SpecManagementServiceImpl;
+import org.kie.server.controller.impl.storage.FileBasedKieServerTemplateStorage;
+import org.kie.server.controller.impl.storage.InMemoryKieServerControllerStorage;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Path("/standalone")
+public class StandaloneController extends KieServerControllerImpl {
```

Comment:
we should use the default RestKieServerContollerImpl and they should be mapped to the same urls - the only thing that should differ is the context path of the app - which is then in hands of users. So any client code written now towards workbench's controller will work here.